plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -1241,6 +1241,7 @@ private void postCreatePrivateTemplate(String installFullPath, long templateId,
             ManagedObjectReference morPool = hyperHost.getHyperHostOwnerResourcePool();
             vmMo.createFullCloneWithSpecificDisk(templateUniqueName, dcMo.getVmFolder(), morPool, VmwareHelper.getDiskDeviceDatastore(volumeDeviceInfo.first()), volumeDeviceInfo);
             clonedVm = dcMo.findVm(templateUniqueName);
+            s_logger.debug(String.format("Cloned VM: %s as %s", vmMo.getName(), clonedVm.getName()));
```

Comment:
@shwstppr can you move this log before this return stmt (in _createFullCloneWithSpecificDisk() @ VirtualMachineMO_):

https://github.com/apache/cloudstack/blob/04738d7775fac382250932f44c7e2cb44a54617c/vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java#L782

Reply:
Done